portfolio/src/main/webapp/script.js

```diff
@@ -48,10 +48,12 @@ function showCommentsForm() {
 /**
  * Fetches a list of comments and displays them on the UI.
  */
-function getComments(language) {
-  fetch('/data?' + new URLSearchParams({lang: language}))
+function getComments() {
+  let language = document.getElementById('languages').value;
+  let commentsContainer = document.getElementById('comments-container');
```

Comment:
const instead of let, use const whenever possible.

Reply:
Done.